airflow/cli/commands/webserver_command.py

```diff
@@ -195,15 +197,11 @@ def webserver(args):
                 port=args.port, host=args.hostname,
                 ssl_context=(ssl_cert, ssl_key) if ssl_cert and ssl_key else None)
     else:
-        os.environ['SKIP_DAGS_PARSING'] = 'True'
-        app = cached_app(None)
-        pid, stdout, stderr, log_file = setup_locations(
+        pid_file, stdout, stderr, log_file = setup_locations(
             "webserver", args.pid, args.stdout, args.stderr, args.log_file)
-        os.environ.pop('SKIP_DAGS_PARSING')
```

Comment:
The `app` was unused and I see no reason for those env changes when we remove the app.

Reply:
App was unused, but this pre-warms the cache, and makes possible errors get reported earlier (i.e. before demonization) so I am not sure about this

Reply:
Reverted the change and added a comment why it's there